gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1493,6 +1493,7 @@ protected void configureConversionUtilities(Project project, SourceSet sourceSet
       }
       task.setKeepOriginal(keepOriginal);
       task.setSkipVerification(skipVerification);
+      task.setEnableArgFile(true);
```

Comment:
Are you sure you want to globally enable this feature? Evan and I reserved it as off by default, and allowing consumers with javaexec issues to opt-in in their build.gradle files.

Reply:
+1 on this. We've seen before that `gradle-plugins` and `restli-tools` are not always locked to the same version. Kyle and I think it's better to keep it opt-in for now, then make it enabled by default later (or if/when we create newer more resilient tools)

Reply:
Done

Reply:
Actually, after thinking about this a little harder, I've realized that: `convertXXXToPdl` and `reformatXXXPdl` are not part of the normal build process, so enabling arg file by default may not be that risky. However, the same cannot be said for `generateXXXDataTemplate` or `XXXTranslateSchemas`; these should be opt-in IMO.